### ISSUE TITLE
Regenerate MoveIt! config files

### ISSUE THREAD

**Issue by jsbruglie** [vislab-tecnico-lisboa/vizzy#64]
Following the changes made in the `gazebo-performance` branch, since the urdf has been changed, the MoveIt! package needs to be regenerated before it can work once again.
Preferably we could have two versions: one which uses a simplified model of the hand and another with a more detailed mesh for collision (maybe even capable of grasping experiments).
As of right now, the detailed hand is not yet implemented, but the arm description file has an empty slot where it should be instanced.

**Comment by science-code** [comment]
@jsbruglie was there any progress on this issue?

**Comment by jsbruglie** [comment]
Not really. This seems easy to do, but there is some older code using MoveIt! which I am unsure how to test. As some parts of the model have changed (even though the name of the joints was not changed) it is possible that some of these demos break. In any case, as of right now the MoveIt! package no longer works with the current main branch URDF and is not being used.
